crates/api_models/src/health_check.rs

```diff
@@ -9,6 +9,8 @@ pub struct RouterHealthCheckResponse {
     #[cfg(feature = "olap")]
     pub opensearch: bool,
     pub outgoing_request: bool,
+    #[cfg(feature = "dynamic_routing")]
+    pub grpc_health_check: bool,
```

Comment:
instead, this should be a map of services that should be healthy right?

Reply:
We only had one service so did not include it atm.
You mean a map of (service -> bool) right?